config/routes.rb

```diff
@@ -10,6 +10,7 @@
     get '/home', to: 'users#home'
     resources :users
     resources :categories
+    resources :words
```

Comment:
We need to loop the routes here. You need to have like this
resources :categories do
      resources :words
end